src/c-writer.cc

```diff
@@ -2171,6 +2188,30 @@ void CWriter::Write(const SimdLaneOpExpr& expr) {
   PushType(result_type);
 }
 
+void CWriter::Write(const SimdLoadLaneExpr& expr) {
+  assert(module_->memories.size() == 1);
+  Memory* memory = module_->memories[0];
+  Type result_type = expr.opcode.GetResultType();
+  Write(StackVar(0, result_type), " = ", expr.opcode.GetName(), "(",
+        ExternalPtr(memory->name), ", (u64)(", StackVar(1), "), ");
+  if (expr.offset != 0)
+    Write(" + ", expr.offset, "u");
+  Write("< ", StackVar(0), ", lane Imm: %d", expr.val, ");", Newline());
```

Comment:
I'm not sure how much of this code is functional atm. This `, lane Imm: ` stuff (already existing before your PR) doesn't seem like it can be valid C. So I assume in addition to not being tested that this code is wip?